tracer/test/test-applications/regression/Sandbox.ManualTracing/Program.cs

```diff
@@ -42,26 +43,32 @@ public static int Main(string[] args)
             }
         }
 
-        public static void RunWebRequestSync()
+        public static void CreateManualTraces()
         {
-            for (int i = 0; i < 5; i++)
+            // Create a configuration source so we can enable runtime metrics without relying on environment variables,
+            // which would require an additional permission to be added
+            var configurationSource = new NameValueConfigurationSource(new NameValueCollection()
```

Comment:
Could you skip the source entirely?
```csharp
var tracerSettings = new TracerSettings { RuntimeMetricsEnabled = true };
```

Reply:
Ideally I would do that, but I can't because the getter/setter for `RuntimeMetricsEnabled` is internal. This seemed like an ok workaround